db/models/index.js

```diff
@@ -8,6 +8,16 @@ const Workout = require('./workout');
 
 // -=-=-=-=-=-= ASSOCIATIONS =-=-=-=-=-=-
 // http://docs.sequelizejs.com/en/latest/docs/associations/
+Therapist.hasMany(Patient) // patient << therapist_id // therapist.getPatients/setPatients
+Exercise.belongsTo(Therapist, { foreignKey: 'therapist_id' }) // exercise << therapist_id
 
+Patient.hasMany(Workout) // workout << patient_id // patient.getWorkouts/setWorkouts
+Plan.hasMany(Workout) // workout << plan_id // plan.getWorkouts/setWorkouts
+
+Patient.hasMany(Treatment) // treatment << patient_id // patient.getTreatments/setTreatments
```

Comment:
Shouldn't this be `Plan.hasMany(Treatment)` as we want to get all the treatments in a specific plan?